openfaas-incubator/vcenter-connector
# Add secret support for credentials

When flags are not given for username/password, the code will now
read this from Kubernetes secrets on disk.

Fixes #25

## Testing

Building the Go code.

@mikejwhitehead PTAL?

You'll need to:
* build an image or use mine
* create your Kubernetes secret (1 secret, two literals / files)
* edit your Kubernetes YAML to bind the two secrets (see how we do it for basic auth)

Signed-off-by: Alex Ellis <alexellis2@gmail.com>
